.github/workflows/analysis.yml

```diff
@@ -2,7 +2,8 @@ name: analysis
 on: [push, pull_request]
 
 jobs:
-  package-analysis:
+  analyze:
```

Comment:
I would maybe name this `analysis`, just like the filename!?

Reply:
Done more changes - changed file names completely - so that can have more than 1 job in a file - file/action name should suggest - when this should happen - instead of what (job name should be based on what).